UnitsNet.Tests/CustomCode/AbsorbedDoseOfIonizingRadiationTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class AbsorbedDoseOfIonizingRadiationTests : AbsorbedDoseOfIonizingRadiationTestsBase
+    {
+        // Override properties in base class here
+        protected override bool SupportsSIUnitSystem => true;
+        protected override double CentigraysInOneGray => 100;
+        protected override double FemtograysInOneGray => 1E+15;
+        protected override double GigagraysInOneGray => 1E-9;
+        protected override double GraysInOneGray => 1;
+        protected override double KilograysInOneGray => 1E-3;
+        protected override double KiloradsInOneGray => 0.1;
+        protected override double MegagraysInOneGray => 1E-06;
+        protected override double MegaradsInOneGray => 1E-4;
+        protected override double MicrograysInOneGray => 1E+6;
+        protected override double MilligraysInOneGray => 1E+3;
+        protected override double NanograysInOneGray => 1E+9;
+        protected override double PetagraysInOneGray => 1E-15;
+        protected override double PicograysInOneGray => 1E+12;
+        protected override double RadsInOneGray => 100;
+        protected override double TeragraysInOneGray => 1E-12;
```

Comment:
Verified these

Reply:
What do you mean? Write some additional tests?

Reply:
No just a note to myself and any other reviewers, that I have verified these constants 👍